static-instrumenter/agent-instrumenter/src/main/java/io/opentelemetry/contrib/statical/instrumenter/StreamUtils.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.statical.instrumenter;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+final class StreamUtils {
+
+  private StreamUtils() {}
+
+  public static void copy(InputStream in, OutputStream out) throws IOException {
```

Comment:
Unless anyone knows people with very legacy build systems will want to use this tool, I would target Java 11 for this tool at least. I'd be in favor of Java 17 to "move the ecosystem forward" too if other maintainers are comfortable with it, but Java 8 seems a bit much.

i.e., we would be able to use `transferTo`

Reply:
Yeah, I sticked to Java8 as it's general OTEL compat... I'd be in favour of going to at least 11 if possible. @open-telemetry/java-contrib-maintainers what do you say here?

Reply:
I don't mind moving to Java 11 for this component. We can always change course in the future if we see significant user pushback (we have no users of it so far, so it's a bit hard to judge).

I think Java 17 is definitely too soon, given how much work it is for some projects to update their builds to Java 17 😉

Reply:
Done!